doc/src/reference/vscodeext-reference-settings.qdoc

```diff
@@ -14,6 +14,28 @@
     To view and change Qt settings, go to \uicontrol {Command Palette} and select
     \uicontrol {Qt: Open Qt Settings}.
 
+    \section1 Use variables in settings
```

Comment:
Could you also mention the `substitution` word in the documentation? That would make it easier for users to find this document. Maybe `variables reference` too.

https://code.visualstudio.com/docs/reference/variables-reference
https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/cmake-settings.md#variable-substitution